rpc-server/src/dispatchers/validator.rs

```diff
@@ -57,4 +57,15 @@ impl ValidatorInterface for ValidatorDispatcher {
         log::debug!("Automatic reactivation set to {}.", automatic_reactivate);
         Ok(().into())
     }
+
+    async fn get_is_validator_elected(&mut self) -> RPCResult<bool, (), Self::Error> {
```

Comment:
`get_is_…` sounds a bit weird to me, I'd prefer `is_…`. Looking at the functions we already expose in the RPC, I can see a couple of both variants.

Reply:
@viquezclaudio please decide, or find someone to decide which naming convention to use.

Reply:
We briefly talked about it, and decided to rename everything to `is_…`: #2181.

Reply:
@sisou Could you make the change and rebase the branch?

Reply:
Done :+1: